tests/test_fft.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+import dpnp
+
+import numpy
+
+@pytest.mark.parametrize("type",
+                         [numpy.complex128, numpy.complex64, numpy.float32, numpy.float64, numpy.int32, numpy.int64],
+                         ids=['complex128', 'complex64', 'float32', 'float64', 'int32', 'int64'])
+def test_fft(type):
+    # 1 dim array
+    data = numpy.arange(100, dtype=type)
+    # TODO:
+    # doesn't work correct with `complex64` (not supported)
+    # dpnp_data = dpnp.arange(100, dtype=type)
+    dpnp_data = dpnp.array(data)
+
+    np_res = numpy.fft.fft(data)
+    dpnp_res = dpnp.fft.fft(dpnp_data)
+
+    numpy.testing.assert_array_equal(dpnp_res, np_res)

```

Comment:
```suggestion
    numpy.testing.assert_allclose(dpnp_res, np_res, rtol=1e-4, atol=1e-7)
```
you can not compare floating point numbers as binary after complex floating point operations. You always need to use  rtol/atol

Reply:
Of course you are right. Floating point numbers are not compared this way.
I specified the function incorrectly.  =)
It seemed to me that `assert_array_equal` works like `assert_allclose` with parameters `rtol = 1e-07, atol = 0` for floating point numbers.
Thank you!